css-text/Overview.bs

```diff
@@ -1296,7 +1299,7 @@ Line Breaking Details</h3>
 
 <pre class='propdef'>
 Name: overflow-wrap, word-wrap
-Value: normal | break-word
+Value: normal | break-word | break-spaces
```

Comment:
@kojiishi ping. Do you want me to adjust something in the pull request before you can merge it, or is it fine like this?